src/main/java/com/limechain/runtime/hostapi/MiscellaneousHostFunctions.java

```diff
@@ -70,4 +166,44 @@ private static Level getLogLevel(int i) {
             default -> Level.FINEST;
         };
     }
+
+    /**
+     * Returns the max logging level used by the host.
+     * <p>
+     * Levels are as follows:
+     * Error = 1
+     * Warn = 2
+     * Info = 3
+     * Debug = 4
+     * Trace = 5
+     * </p>
+     * We have Severe, Warning, Info, Fine and Finest
+     * </p>
+     * If we map
+     * Severe -> Error
+     * Warning -> Warn
+     * Info -> Info
+     * Fine -> Debug
+     * Finest -> Trace
+     * </p>
+     * our max level is 5. (The returned value is the index of the level - 4)
+     *
+     * @return the max log level used by the host.
+     */
+    public int maxLevelV1() {
+        return 4;
+    }
+
+    /**
+     * Aborts the execution of the runtime with a given message.
+     *
+     * @param messagePtr a pointer-size to the UTF-8 encoded message.
+     */
+    public void extPanicHandlerAbortOnPanicVersion1(RuntimePointerSize messagePtr) {
+        byte[] data = hostApi.getDataFromMemory(messagePtr);
+
+        final String messageToPrint = new String(data, StandardCharsets.UTF_8);
+
+        log.severe(String.format("Aborting runtime panicked with message: %s", messageToPrint));
```

Comment:
Should this method abort the execution of the runtime, or just log?

Reply:
The Polkadot spec doesn't give enough details on what to abort. Also, other hosts haven't used this function, so we might be fine if we remove it.

Reply:
Removed in 00d07abeb131e03fe072fb85ce53bac1934a94cf